scripts/zones/Western_Altepa_Desert/mobs/King_Vinegarroon.lua

```diff
@@ -10,6 +10,7 @@ local entity = {}
 
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:setMobMod(xi.mobMod.DRAW_IN, 2)
```

Comment:
Going forward, I think it would be useful to mark what these values do:
`mob:setMobMod(xi.mobMod.DRAW_IN, 2) -- Alliance Draw-In` etc.